lte/gateway/python/magma/subscriberdb/store/sqlite.py

```diff
@@ -266,6 +283,40 @@ def resync(self, subscribers):
                 conn.close()
         self._on_ready.resync(subscribers)
 
+    def get_current_digest(self):
+        """
+        Method that returns the current subscriber digest stored in the db.
+        """
+        conn = sqlite3.connect(self._digest_db_location, uri=True)
+        try:
+            with conn:
+                res = conn.execute(
+                    "SELECT * FROM subscriber_digest",
```

Comment:
As a general rule: don't use the blanket select (`SELECT *`) in application code except with specific reason -- (a) for readability and (b) won't break [if the columns ever got reordered](https://stackoverflow.com/questions/6805426/how-to-move-columns-in-a-mysql-table)